src/client/schedules/types.ts

```diff
@@ -42,6 +42,30 @@ export type ScheduleReadOptions = {
 	omit?: string[]
 }
 
+/**
+ * Options for reading a Schedules events.
+ */
+export type ScheduleEventsReadOptions = {
+	start: string
+	end?: string
+}
+
+/**
+ * The shape of an event on a Schedule.
+ */
+interface ScheduleEvent extends HDict {
+	startDate: HDateTime
```

Comment:
Properties returned are named `startTime` and `endTime`

Reply:
Good catch! Updated.